src/ITest/java/uk/gov/companieshouse/pscstatementdataapi/steps/PscStatementsSteps.java

```diff
@@ -256,6 +258,11 @@ public void statement_exists(String statementId) {
         Assertions.assertThat(pscStatementRepository.existsById(statementId)).isTrue();
     }
 
+    @When("no statement exists with id {string}")
+    public void no_statement_exists(String statementId) {
+        Assertions.assertThat(pscStatementRepository.existsById(statementId)).isTrue();
+    }
+
```

Comment:
Should this not be isFalse? Also it's more of an assertion, should it be a Then step??